CMakeLists.txt

```
@@ -99,7 +98,7 @@ target_sources(gtracers PRIVATE
 
 target_link_libraries(gtracers PUBLIC
 			FMS::fms_r8
-			PkgConfig::MOCSY)
+		  mocsy::mocsy)
```

Sorry, one little whitespace change

```suggestion
			mocsy::mocsy)
```